speechbrain/decoders/seq2seq.py

```diff
@@ -15,6 +15,9 @@
     _update_mem,
 )
 from speechbrain.utils.data_utils import undo_padding
+import torch.nn.functional as F
```

Comment:
I know this is a typical import for pytorch but I tend to avoid it if I can, `F` is pretty vague